src/Features/Core/Portable/CodeFixes/PredefinedCodeFixProviderNames.cs

```diff
@@ -28,6 +28,7 @@ internal static class PredefinedCodeFixProviderNames
         public const string ImplementInterface = "Implement Interface";
         public const string InsertMissingCast = "InsertMissingCast";
         public const string MoveToTopOfFile = "Move To Top Of File";
+        public const string QualifyMemberAccess = "Qualify member access";
```

Comment:
Pretty sure the names are only used for extension ordering attributes. If they need to be human-readable then we have a loc bug, but I don't think they do. Please verify the Name isn't used anywhere and use Cyrus's ```nameof``` suggestion or file a loc bug with the appropriate Tenet-* labels.